scale-decode/Cargo.lock

```diff
@@ -0,0 +1,235 @@
+# This file is automatically @generated by Cargo.
```

Comment:
why is this file committed?

Reply:
Oh good catch; it shouldn't be! I guess my gitignore wasn't good enough!

Reply:
Deleted it; I think it was from when I originally moved all of this stuff into a sub crate and forgot to remove the lock file!